api/repr/impl.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package repr
+
+import (
+	"github.com/GoogleCloudPlatform/heapster/sinks/cache"
+	"github.com/golang/glog"
+	"time"
+)
+
+func NewCluster() Cluster {
+	var zero time.Time
```

Comment:
Time should be zero by default. We can skip updating it and just fill in the cinfo below.